packages/e2e/features/ui/components/authenticator/sign-up-with-email.feature

```diff
@@ -12,9 +12,9 @@ Feature: Sign Up with Email
     And I don't see "Username" as an input field
     And I don't see "Phone Number" as an input field
 
-  # Sign up tests skipped due to SES limits
-  Scenario: Sign up with valid email & password
-    When I type my "email" with status "UNCONFIRMED"
+  @todo-angular @react @todo-vue
```

Comment:
Should this be
```suggestion
  @angular @react @vue @todo-angular  @todo-vue
```
if we want to run this on `main`?

Reply:
Oh you're right! `@angular` for support & `@todo-angular` to indicate that it's a known gap.